analyzer/analyzer.go

```diff
@@ -91,20 +91,17 @@ func readRules() (*parseRulesResult, error) {
 
 	switch {
 	case flagRules != "":
-		if flagRules == "" {
-			return nil, fmt.Errorf("-rules values is empty")
-		}
 		filenames := strings.Split(flagRules, ",")
 		var ruleSets []*ruleguard.GoRuleSet
 		for _, filename := range filenames {
 			filename = strings.TrimSpace(filename)
-			data, err := ioutil.ReadFile(filename)
-			if err != nil {
-				return nil, fmt.Errorf("read rules file: %v", err)
+			loader := loadLocalConfig
+			if strings.HasPrefix(filename, "http://") || strings.HasPrefix(filename, "https://") {
+				loader = loadRemoteConfig
 			}
```

Comment:
this routine can be hided inside a func

Reply:
and this also will simplify config loader

Reply:
Fixed :+1: